Jellyfin/Resources/winuwp.js

```diff
@@ -0,0 +1,140 @@
+        'externallinks',
+        'htmlaudioautoplay',
+        'htmlvideoautoplay',
+        'multiserver',
+        'otherapppromotions',
+        'screensaver',
+        'subtitleappearancesettings',
+        'subtitleburnsettings',
+        'targetblank'
+    ];
+
+    if (xbox || mobile) {
+        SupportedFeatures.push('physicalvolumecontrol');
+    }
+
+    // Only devices with HdmiDisplayInformation have settings currently
+    if (xbox) {
+        SupportedFeatures.push('clientsettings');
+    }
+
+    console.debug('SupportedFeatures', SupportedFeatures);
+
+    window.NativeShell = {
+        AppHost: {
+            init: function () {
+                console.debug('AppHost.init', AppInfo);
+                return Promise.resolve(AppInfo);
+            },
+
+            appName: function () {
+                console.debug('AppHost.appName', AppInfo.appName);
+                return AppInfo.appName;
+            },
+
+            appVersion: function () {
+                console.debug('AppHost.appVersion', AppInfo.appVersion);
+                return AppInfo.appVersion;
+            },
+
+            deviceName: function () {
+                console.debug('AppHost.deviceName', AppInfo.deviceName);
+                return AppInfo.deviceName;
+            },
+
+            exit: function () {
+                postMessage('exit');
+            },
+
+            getDefaultLayout: function () {
+                let layout;
+                if (xbox) {
+                    layout = 'tv';
+                } else if (mobile) {
+                    layout = 'mobile';
+                } else {
+                    layout = 'desktop';
+                }
+                console.debug('AppHost.getDefaultLayout', layout);
+                return layout;
+            },
+
+            getDeviceProfile: function (profileBuilder) {
+                console.debug('AppHost.getDeviceProfile');
+                const options = {};
+                if (supportsHdr10 != null) {
+                    options.supportsHdr10 = supportsHdr10;
+                }
+                if (supportsDolbyVision != null) {
+                    options.supportsDolbyVision = supportsDolbyVision;
+                }
+                if (xboxSeries) {
+                    options.maxVideoWidth = 3840;
+                }
+                return profileBuilder(options);
+            },
+
+            supports: function (command) {
+                const isSupported = command && SupportedFeatures.indexOf(command.toLowerCase()) !== -1;
+                console.debug('AppHost.supports', {
+                    command: command,
+                    isSupported: isSupported
+                });
+                return isSupported;
+            }
+        },
+
+        enableFullscreen: function (videoInfo) {
+            postMessage('enableFullscreen', videoInfo);
+        },
+
+        disableFullscreen: function () {
+            postMessage('disableFullscreen');
```

Comment:
Leaving fullscreen isn't supported on devices that utilise the `hevcPlayback` capbilities